notes-with-indexeddb/background.js

```diff
@@ -0,0 +1,14 @@
+/*
+Open a new tab, and load "notes.html" into it.
+*/
+function openNotes() {
+    chrome.tabs.create({
+        "url": chrome.extension.getURL("notes.html")
```

Comment:
The `chrome.extension` namespace is being phased out, so please use `runtime.getURL` instead.